_posts/2022-08-15-Remote-dev-watch.adoc

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Remote dev-watch development with WildFly Jar Maven Plugin
+date: 2022-08-15
+tags: wildfly bootable jar bootable-jar openshift cloud remote
+synopsis: Bootable JAR Maven plugin 8.0.0.Alpha1 brings in a new way to make remote development easier.
+author: ehsavoie
+---
+
+The 8.0.0.Alpha1 version of the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/[WildFly bootable JAR Maven plugin] has been released.
+
+This  https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/tag/8.0.0.Alpha1release] is not yet Final, as it is only there to gather feedback on a new feature to simplify develoment on the 'cloud' using the `dev-watch` goal.
```

Comment:
Wrongly formatted link
"to simplify develoment" ==> that simplifies development